forge-gui/res/cardsfolder/upcoming/astarions_thirst.txt

```diff
@@ -0,0 +1,9 @@
+Name:Astarion's Thirst
+ManaCost:3 B
+Types:Instant
+A:SP$ ChangeZone | ValidTgts$ Creature | TgtPrompt$ Select target creature | Origin$ Battlefield | Destination$ Exile | RememberChanged$ True | SubAbility$ DBPutCounter | SpellDescription$ Exile target creature. Put X +1/+1 counters on a commander creature you control, where X is the power of the creature exiled this way.
+SVar:DBPutCounter:DB$ PutCounter | Choices$ Creature.YouCtrl+IsCommander | CounterType$ P1P1 | CounterNum$ X | SubAbility$ DBCleanup
```

Comment:
How does the StackDescription look for this on cast?

Reply:
The current script produces this stack description:

"Exile {targeted}. they puts zero +1/+1 counters on a Creature.YouCtrl+IsCommander."

I took some time testing the StackDescriptions further, changing the end of line 4 to the following:

`StackDescription$ Exile {c:Targeted}. | SpellDescription$ StackDescription`

And adding a StackDescription to line 5 as follows:

`StackDescription$ Put X +1/+1 counters on a commander creature you control, where X is the power of the creature exiled this way.`

With these changes, I arrived at a stack description that reads:

"Exile {targeted}. Put X +1/+1 counters on a commander creature you control."

Is this the end goal we're seeking? Or are we looking to have the stack description say "<player> puts X +1/+1 counters on a commander creature <they> control"?

Reply:
Line 4 should work fine with just `SpellDescription$ Exile CARDNAME.` Forge should be able to parse this without any trouble.
Line 5 is the one Forge would struggle parsing, and I think your solution is fine.

Reply:
Line 5 you probably want:
```
StackDescription$ SpellDescription | SpellDescription$ Put X +1/+1 counters on a commander creature you control, where X is the power of the creature exiled this way.
```

As a general rule you want to check the card detail pane and the stack. The card detail pane should match the oracle, though re-ordering of triggers, statics, etc. is fine. The stack should at least make sense. If you see lots of unparsed output then you may want to reach for the StackDescription or ping me about it.

Reply:
I see what went wrong then, I must have swapped the order of StackDesc$ and SpellDesc$. Appreciate the extra set of eyes